src/filters/type.js

```diff
@@ -108,6 +115,49 @@ export const elide = (html, count = 50) => {
   return truncated;
 };
 
+/* @docs
+label: stripTagsForRSS
+category: RSS
+note: |
+  Remove permalinks from headings
+params:
+  html:
+    type: string
+*/
+export const stripTagsForRSS = async (html) => {
+  const modifier = posthtml().use((tree) => {
+    if (!Array.isArray(tree) || !tree.length) {
+      return tree;
+    }
+    tree.walk((node) => {
+      const classes =
+        node.attrs?.class?.split(' ')?.map((c) => c.trim().toLowerCase()) || [];
+      const isPermalink = node.tag === 'a' && classes.includes('header-anchor');
+      const isUnsafe = [
+        'comment',
+        'embed',
+        'link',
+        'listing',
+        'meta',
+        'noscript',
+        'object',
+        'plaintext',
+        'script',
+        'xmp',
+      ].includes(node.tag);
+      const isCustom = ['is-land', 'lite-youtube'].includes(node.tag);
```

Comment:
@mirisuzanne After running this through the [W3C validator](https://validator.w3.org/feed/check.cgi?url=https://deploy-preview-550--oddleventy.netlify.app/feed.atom#l799), I figured we probably want to strip out these unsafe and custom tags too. I made the change in https://github.com/oddbird/oddleventy/pull/550/commits/82890cb3d63379ae0cd91e79328957e62ee068d8 -- what do you think?

Reply:
That seems fine to me. I guess check the build output to see if anything surprising is missing?

Reply:
Out of `feed.atom` looks fine to me -- just with anchor links, a few <script> tags, and custom elements removed.

@mirisuzanne I think this is ready to merge once you've given final review?

Reply:
@mirisuzanne Do you think this is good to merge?

Reply:
yep, lgtm